libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -29,9 +29,9 @@ void PbftSyncPacketHandler::validatePacketRlpFormat(const PacketData &packet_dat
 
   // PeriodData rlp parsing cannot be done through util::rlp_tuple, which automatically checks the rlp size so it is
   // checked here manually
-  if (constexpr size_t required_size = 4; packet_data.rlp_[1].itemCount() != required_size) {
+  if (packet_data.rlp_[1].itemCount() != PeriodData::kItemCount) {
```

Comment:
kItemsCount

Reply:
actually it should be without s, it is numer_of_items or ItemCount not ItemsCount